derivation-cli/src/main/java/uk/gov/register/derivation/cli/Main.java

```diff
@@ -14,13 +15,16 @@
 public class Main {
     public static void main(String[] args) throws IOException {
         if (args.length == 3) {
-            Injector injector = Guice.createInjector(new Registrar());
+            Injector injector = Guice.createInjector(new DerivationCliModule());
 
             InputStream rsfStream = Files.newInputStream(Paths.get(args[0]));
             RsfParser parser = injector.getInstance(RsfParser.class);
             Set<PartialEntity> entities = parser.parse(rsfStream);
 
-            String jsonResult = JsonSerializer.serialize(entities);
+            RegisterTransformer transformer = injector.getInstance(RegisterTransformer.class);
```

Comment:
I think we should inject `CurrentCountryFilter` rather than `RegisterTransformer` here